doc/architecture/decisions/0002-object-identifiers.md

```diff
@@ -0,0 +1,57 @@
+# 0002. Object identifiers
```

Comment:
I can't see any mention that the dataset identifier should be, in someway, tied to a track to facilitate this lookup